genie-web/src/integTest/resources/application-integration.yml

```diff
@@ -29,9 +29,6 @@ genie:
     cluster-selector:
       source: file:///tmp/genie/scripts/selectCluster.js
       auto-load-enabled: true
-    execution-mode-filter:
```

Comment:
Were these documented as well? If so that documentation should be deleted

Reply:
All of this was deleted in a previous PR: https://github.com/Netflix/genie/commit/491387a33aa7128e2a143c77d955323c76c6b581 these default just got left behind